src/display/KT-LCD3/lcd.c

```diff
@@ -1286,56 +1299,6 @@ void lcd_execute_menu_config_submenu_various (void)
         lcd_print (ui8_temp, ODOMETER_FIELD, 1);
       }
     break;
-
-    // PAS max cadence
```

Comment:
Was this accidentally removed? PAS max cadence is still used but can no longer be configured...

Reply:
Removed pas_max_cadence